triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -60,11 +60,12 @@ object TriggerServiceFixture {
       mat: Materializer,
       aesf: ExecutionSequencerFactory,
       ec: ExecutionContext): Future[A] = {
-    // Launch a toxiproxy instance . Wait on it to be ready to accept
+    // Launch a toxiproxy instance. Wait on it to be ready to accept
     // connections.
     val host = InetAddress.getLoopbackAddress()
-    val toxiProxyExe = BazelRunfiles.rlocation("external/toxiproxy_nix/bin/toxiproxy-cmd")
-    val toxiProxyPort = findFreePort()
+    val toxiProxyExe = BazelRunfiles.rlocation("external/toxiproxy_dev_env/toxiproxy")
```

Comment:
if it still fails, maybe try adding a `.exe` to the end on Windows or maybe use backslashes.